server/controllers/documents.js

```diff
@@ -0,0 +1,289 @@
+import helper from '../helpers/helper';
+import models from '../models/index';
+
+/**
+   * Creates a new instance of document in the database
+   * @function createDocument
+   * @param {object} request request
+   * @param {object} response response
+   * @return {object} object - information about the request
```

Comment:
isn't this supposed to be a response object. I don't think information about the request is descriptive enough